scripts/babel-relay-plugin/src/babelAdapter.js

```diff
@@ -26,7 +26,25 @@ function babelAdapter(
 ): mixed {
   if (Plugin == null) {
     // Babel 6.
-    return visitorsBuilder(t);
+    const { visitor: { Program, TaggedTemplateExpression } } = visitorsBuilder(t);
+
+    const taggedTemplateExpressionVisitor = {
+      TaggedTemplateExpression(path) {
+        return TaggedTemplateExpression(path, this);
```

Comment:
So, the way I understand it is that `TaggedTemplateExpression`, when in it's normal plugin form, takes in `path` and `state` as arguments. `state` doesn't get passed in, I don't think, when using `path.traverse`. Instead, state is passed in as `this`.

Honestly, I'm not a super Babel expert either -- I followed what @thejameskyle did here:

https://phabricator.babeljs.io/rBC0b264aa38af5ba7fb121b216be2fb69eece7217c